## 94% of end-end tests have passed
```
Total executed tests: 51
Failed tests: 3
Expected to fail tests: 0
Passed tests: 48
```
```
IDs of failed tests: 703495,702839,702838 
```
<h3>Failed tests (3)</h3><details><summary>Click to expand</summary><br/><li><a href="https://ci.status.im/job/status-mobile/job/e2e/job/status-app-prs-rerun/parambuild/?BRANCH_NAME=fix/collectibles-fetching-for-newly-added-accounts&PR_ID=20961&APK_NAME=20961.apk&TR_CASE_IDS=703495,702839,702838">Rerun failed tests</a></li><br/><h4>Class TestCommunityMultipleDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3646511">test_community_message_delete</a>, id: 702839 </b></td></tr><tr><td><p><blockquote><div>Device 2: Find `Text` by `xpath`: `//android.view.ViewGroup[@content-desc='chat-item']//android.widget.TextView[contains(@text,'delete for everyone')]` </div><div>Device 2: Long press on `Text` </div></blockquote></p>

```
critical/chats/test_public_chat_browsing.py:391: in test_community_message_delete
    self.channel_2.delete_message_in_chat(message_to_delete_everyone)
../views/chat_view.py:1028: in delete_message_in_chat
    self.chat_element_by_text(message).message_body.long_press_element()
../views/base_element.py:324: in long_press_element
    action.long_press(element).release().perform()
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/appium/webdriver/common/touch_action.py:174: in perform
    self._driver.execute(Command.TOUCH_ACTION, params)
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:345: in execute
    self.error_handler.check_response(response)
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/appium/webdriver/errorhandler.py:122: in check_response
    raise exception_class(msg=message, stacktrace=format_stacktrace(stacktrace))
 The element 'By.xpath: //android.view.ViewGroup[@content-desc='chat-item']//android.widget.TextView[contains(@text,'delete for everyone')]' is not linked to the same object in DOM anymore; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#stale-element-reference-exception
E   Stacktrace:
E   io.appium.uiautomator2.common.exceptions.StaleElementReferenceException: The element 'By.xpath: //android.view.ViewGroup[@content-desc='chat-item']//android.widget.TextView[contains(@text,'delete for everyone')]' is not linked to the same object in DOM anymore
E   	at io.appium.uiautomator2.model.ElementsCache.restore(ElementsCache.java:122)
E   	at io.appium.uiautomator2.model.ElementsCache.get(ElementsCache.java:153)
E   	at io.appium.uiautomator2.handler.Location.safeHandle(Location.java:23)
E   	at io.appium.uiautomator2.handler.request.SafeRequestHandler.handle(SafeRequestHandler.java:59)
E   	at io.appium.uiautomator2.server.AppiumServlet.handleRequest(AppiumServlet.java:277)
E   	at io.appium.uiautomator2.server.AppiumServlet.handleHttpRequest(AppiumServlet.java:271)
E   	at io.appium.uiautomator2.http.ServerHandler.channelRead(ServerHandler.java:68)
E   	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:366)
E   	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:352)
E   	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:345)
E   	at io.netty.handler.codec.MessageToMessageDecoder.channelRead(MessageToMessageDecoder.java:102)
E   	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:366)
E   	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:352)
E   	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:345)
E   	at io.netty.channel.CombinedChannelDuplexHandler$DelegatingChannelHandlerContext.fireChannelRead(CombinedChannelDuplexHandler.java:435)
E   	at io.netty.handler.codec.ByteToMessageDecoder.fireChannelRead(ByteToMessageDecoder.java:293)
E   	at io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:267)
E   	at io.netty.channel.CombinedChannelDuplexHandler.channelRead(CombinedChannelDuplexHandler.java:250)
E   	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:366)
E   	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:352)
E   	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:345)
E   	at io.netty.handler.timeout.IdleStateHandler.channelRead(IdleStateHandler.java:266)
E   	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:366)
E   	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:352)
E   	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:345)
E   	at io.netty.channel.DefaultChannelPipeline$HeadContext.channelRead(DefaultChannelPipeline.java:1294)
E   	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:366)
E   	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:352)
E   	at io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:911)
E   	at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:131)
E   	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:611)
E   	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:552)
E   	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:466)
E   	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:438)
E   	at io.netty.util.concurrent.SingleThreadEventExecutor$2.run(SingleThreadEventExecutor.java:140)
E   	at io.netty.util.concurrent.DefaultThreadFactory$DefaultRunnableDecorator.run(DefaultThreadFactory.java:144)
E   	at java.lang.Thread.run(Thread.java:1012)
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/f861f77e3c224083b3698e96e3dd160a?auth=74ab0b05a73f42f7177711f7a9787e36#206">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/b5d795fac99442beab307f49935b8a20?auth=db4f6351bdaf4e6eef765fe27163e8b4#140">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3646510">test_community_message_send_check_timestamps_sender_username</a>, id: 702838 </b></td></tr><tr><td><p><blockquote><div>Device 2: Verifying that 'new message' is under today </div><div>Device 2: Looking for a message by text: new message </div></blockquote></p>

```
critical/chats/test_public_chat_browsing.py:363: in test_community_message_send_check_timestamps_sender_username
    self.channel_2.verify_message_is_under_today_text(new_message, self.errors, 60)
../views/chat_view.py:971: in verify_message_is_under_today_text
    message_element.wait_for_visibility_of_element(timeout)
../views/base_element.py:147: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: ChatElementByText by xpath:`//*[starts-with(@text,'new message')]/ancestor::android.view.ViewGroup[@content-desc='chat-item']` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/f861f77e3c224083b3698e96e3dd160a?auth=74ab0b05a73f42f7177711f7a9787e36#164">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/b5d795fac99442beab307f49935b8a20?auth=db4f6351bdaf4e6eef765fe27163e8b4#109">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestGroupChatMultipleDeviceMergedNewUI:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3646535">test_group_chat_mute_chat</a>, id: 703495 </b></td></tr><tr><td><p><blockquote><div>Device 2: Click until `ChatMessageInput` by `accessibility id`: `chat-message-input` will be presented </div><div>Device 2: Looking for a message by text: Chat is unmuted now </div></blockquote></p>

```
critical/chats/test_group_chat.py:520: in test_group_chat_mute_chat
    self.errors.verify_no_errors()
base_test_case.py:191: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Text 'Muted until 00:01 today' is not shown for muted chat
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d3884fd1bfb44ae480a724253f00c8de?auth=c8713c5bf91e0ffe60ae4cf1b21ad0bf#477">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/78c29d665fdd410d9bcb3a356fa80d26?auth=e1ad165f2d369fea93a9f5ae302a759c#279">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/a68a9ea6fd1644568bdd2e15988f5a8e?auth=a0e8819196a84937af3b5d6b4859561c#248">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details><h3>Passed tests (48)</h3><details><summary>Click to expand</summary><br/><br/><h4>Class TestOneToOneChatMultipleSharedDevicesNewUiTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3646504">test_1_1_chat_delete_via_long_press_relogin</a>, id: 702784 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c6949c371a5a4209a105ef90ca2a7692?auth=04eda35fdbb7b23f6d81b4b44b4de9b3#184">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/7370257264c746a58f8b64a66fd850f3?auth=5160efa21e312634ed2211c249a5498c#131">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3646503">test_1_1_chat_is_shown_message_sent_delivered_from_offline</a>, id: 702783 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c6949c371a5a4209a105ef90ca2a7692?auth=04eda35fdbb7b23f6d81b4b44b4de9b3#107">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/7370257264c746a58f8b64a66fd850f3?auth=5160efa21e312634ed2211c249a5498c#90">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3646536">test_1_1_chat_mute_chat</a>, id: 703496 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c6949c371a5a4209a105ef90ca2a7692?auth=04eda35fdbb7b23f6d81b4b44b4de9b3#117">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/7370257264c746a58f8b64a66fd850f3?auth=5160efa21e312634ed2211c249a5498c#118">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestActivityMultipleDevicePR:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3646524">test_activity_center_reply_read_unread_delete_filter_swipe</a>, id: 702947 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/78ca5e518e994265bd7f24817f59ed0f?auth=8cd010121ec89fe139ae32838039b9e9#129">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/877c9350752f42b38f8bd136f3dcbcec?auth=4ef3346fc5df2e3860106dd4eaa8698d#99">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestDeepLinksOneDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3646539">test_links_open_universal_links_from_chat</a>, id: 704613 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/1813ced012b048de9c01b0f69082895f?auth=3f96d258ad8ad4bd52373369d6cd86bd#61">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3646500">test_links_deep_links_profile</a>, id: 702775 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/1813ced012b048de9c01b0f69082895f?auth=3f96d258ad8ad4bd52373369d6cd86bd#124">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3646544">test_deep_links_communities</a>, id: 739307 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/1813ced012b048de9c01b0f69082895f?auth=3f96d258ad8ad4bd52373369d6cd86bd#194">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestOneToOneChatMultipleSharedDevicesNewUi:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3646502">test_1_1_chat_emoji_send_reply_and_open_link</a>, id: 702782 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/929404061acb486daba0a5ec265a90d4?auth=35bbed84cb1b1ecaedd4eec5ca44d81a#157">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/82cfd49a9e764295a8a30037673f6491?auth=39622767bfda43823b67d57ae4cc1e35#125">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3646497">test_1_1_chat_text_message_delete_push_disappear</a>, id: 702733 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/929404061acb486daba0a5ec265a90d4?auth=35bbed84cb1b1ecaedd4eec5ca44d81a#568">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/82cfd49a9e764295a8a30037673f6491?auth=39622767bfda43823b67d57ae4cc1e35#406">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3646509">test_1_1_chat_push_emoji</a>, id: 702813 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/929404061acb486daba0a5ec265a90d4?auth=35bbed84cb1b1ecaedd4eec5ca44d81a#427">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/82cfd49a9e764295a8a30037673f6491?auth=39622767bfda43823b67d57ae4cc1e35#266">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3646499">test_1_1_chat_non_latin_messages_stack_update_profile_photo</a>, id: 702745 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/929404061acb486daba0a5ec265a90d4?auth=35bbed84cb1b1ecaedd4eec5ca44d81a#369">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/82cfd49a9e764295a8a30037673f6491?auth=39622767bfda43823b67d57ae4cc1e35#215">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3646520">test_1_1_chat_edit_message</a>, id: 702855 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/929404061acb486daba0a5ec265a90d4?auth=35bbed84cb1b1ecaedd4eec5ca44d81a#476">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/82cfd49a9e764295a8a30037673f6491?auth=39622767bfda43823b67d57ae4cc1e35#297">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3646534">test_1_1_chat_send_image_save_and_share</a>, id: 703391 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/929404061acb486daba0a5ec265a90d4?auth=35bbed84cb1b1ecaedd4eec5ca44d81a#507">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/82cfd49a9e764295a8a30037673f6491?auth=39622767bfda43823b67d57ae4cc1e35#343">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>7. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3646495">test_1_1_chat_pin_messages</a>, id: 702731 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/929404061acb486daba0a5ec265a90d4?auth=35bbed84cb1b1ecaedd4eec5ca44d81a#223">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/82cfd49a9e764295a8a30037673f6491?auth=39622767bfda43823b67d57ae4cc1e35#170">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>8. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3646494">test_1_1_chat_message_reaction</a>, id: 702730 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/929404061acb486daba0a5ec265a90d4?auth=35bbed84cb1b1ecaedd4eec5ca44d81a#101">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/82cfd49a9e764295a8a30037673f6491?auth=39622767bfda43823b67d57ae4cc1e35#90">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestWalletMultipleDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3646542">test_wallet_send_asset_from_drawer</a>, id: 727230 </b></td></tr><tr><td></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3646541">test_wallet_send_eth</a>, id: 727229 </b></td></tr><tr><td></td></tr></tbody></table><h4>Class TestGroupChatMultipleDeviceMergedNewUI:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3646496">test_group_chat_pin_messages</a>, id: 702732 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d3884fd1bfb44ae480a724253f00c8de?auth=c8713c5bf91e0ffe60ae4cf1b21ad0bf#355">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/78c29d665fdd410d9bcb3a356fa80d26?auth=e1ad165f2d369fea93a9f5ae302a759c#236">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/a68a9ea6fd1644568bdd2e15988f5a8e?auth=a0e8819196a84937af3b5d6b4859561c#234">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3646532">test_group_chat_send_image_save_and_share</a>, id: 703297 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d3884fd1bfb44ae480a724253f00c8de?auth=c8713c5bf91e0ffe60ae4cf1b21ad0bf#282">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/78c29d665fdd410d9bcb3a356fa80d26?auth=e1ad165f2d369fea93a9f5ae302a759c#174">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/a68a9ea6fd1644568bdd2e15988f5a8e?auth=a0e8819196a84937af3b5d6b4859561c#147">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3646531">test_group_chat_reactions</a>, id: 703202 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d3884fd1bfb44ae480a724253f00c8de?auth=c8713c5bf91e0ffe60ae4cf1b21ad0bf#175">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/78c29d665fdd410d9bcb3a356fa80d26?auth=e1ad165f2d369fea93a9f5ae302a759c#112">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/a68a9ea6fd1644568bdd2e15988f5a8e?auth=a0e8819196a84937af3b5d6b4859561c#92">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3646506">test_group_chat_join_send_text_messages_push</a>, id: 702807 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d3884fd1bfb44ae480a724253f00c8de?auth=c8713c5bf91e0ffe60ae4cf1b21ad0bf#158">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/78c29d665fdd410d9bcb3a356fa80d26?auth=e1ad165f2d369fea93a9f5ae302a759c#84">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/a68a9ea6fd1644568bdd2e15988f5a8e?auth=a0e8819196a84937af3b5d6b4859561c#84">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3646507">test_group_chat_offline_pn</a>, id: 702808 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d3884fd1bfb44ae480a724253f00c8de?auth=c8713c5bf91e0ffe60ae4cf1b21ad0bf#322">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/78c29d665fdd410d9bcb3a356fa80d26?auth=e1ad165f2d369fea93a9f5ae302a759c#209">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/a68a9ea6fd1644568bdd2e15988f5a8e?auth=a0e8819196a84937af3b5d6b4859561c#207">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestWalletOneDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3646543">test_wallet_add_remove_regular_account</a>, id: 727231 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9202830c2c344ffabf5314fccdd149f3?auth=df4ddf15953aca0ad11efae45608ca5b#39">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityMultipleDeviceMergedTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3646508">test_community_markdown_support</a>, id: 702809 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/22eeca622312446587f045c3e261e0c4?auth=170c964a2b6f3ed11c9ae0550e0d0a37#186">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/15218bfaa165477b9ba4b4c14e0bf641?auth=d29f93ba34627db03ea01dc889e2fcde#174">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3646525">test_community_hashtag_links_to_community_channels</a>, id: 702948 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/22eeca622312446587f045c3e261e0c4?auth=170c964a2b6f3ed11c9ae0550e0d0a37#305">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/15218bfaa165477b9ba4b4c14e0bf641?auth=d29f93ba34627db03ea01dc889e2fcde#298">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3646505">test_community_mentions_push_notification</a>, id: 702786 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/22eeca622312446587f045c3e261e0c4?auth=170c964a2b6f3ed11c9ae0550e0d0a37#137">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/15218bfaa165477b9ba4b4c14e0bf641?auth=d29f93ba34627db03ea01dc889e2fcde#116">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3646516">test_community_leave</a>, id: 702845 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/22eeca622312446587f045c3e261e0c4?auth=170c964a2b6f3ed11c9ae0550e0d0a37#302">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/15218bfaa165477b9ba4b4c14e0bf641?auth=d29f93ba34627db03ea01dc889e2fcde#276">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3646538">test_community_join_when_node_owner_offline</a>, id: 703629 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/22eeca622312446587f045c3e261e0c4?auth=170c964a2b6f3ed11c9ae0550e0d0a37#504">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/15218bfaa165477b9ba4b4c14e0bf641?auth=d29f93ba34627db03ea01dc889e2fcde#477">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityMultipleDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3646530">test_community_several_images_send_reply</a>, id: 703194 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/f861f77e3c224083b3698e96e3dd160a?auth=74ab0b05a73f42f7177711f7a9787e36#209">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/b5d795fac99442beab307f49935b8a20?auth=db4f6351bdaf4e6eef765fe27163e8b4#151">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3646521">test_community_one_image_send_reply</a>, id: 702859 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/f861f77e3c224083b3698e96e3dd160a?auth=74ab0b05a73f42f7177711f7a9787e36#234">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/b5d795fac99442beab307f49935b8a20?auth=db4f6351bdaf4e6eef765fe27163e8b4#214">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3646512">test_community_emoji_send_copy_paste_reply</a>, id: 702840 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/f861f77e3c224083b3698e96e3dd160a?auth=74ab0b05a73f42f7177711f7a9787e36#282">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/b5d795fac99442beab307f49935b8a20?auth=db4f6351bdaf4e6eef765fe27163e8b4#270">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3646528">test_community_mark_all_messages_as_read</a>, id: 703086 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/f861f77e3c224083b3698e96e3dd160a?auth=74ab0b05a73f42f7177711f7a9787e36#528">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/b5d795fac99442beab307f49935b8a20?auth=db4f6351bdaf4e6eef765fe27163e8b4#452">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3646523">test_community_contact_block_unblock_offline</a>, id: 702894 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/f861f77e3c224083b3698e96e3dd160a?auth=74ab0b05a73f42f7177711f7a9787e36#401">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/b5d795fac99442beab307f49935b8a20?auth=db4f6351bdaf4e6eef765fe27163e8b4#380">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3646540">test_community_edit_delete_message_when_offline</a>, id: 704615 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/f861f77e3c224083b3698e96e3dd160a?auth=74ab0b05a73f42f7177711f7a9787e36#563">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/b5d795fac99442beab307f49935b8a20?auth=db4f6351bdaf4e6eef765fe27163e8b4#478">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>7. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3646515">test_community_links_with_previews_github_youtube_twitter_gif_send_enable</a>, id: 702844 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/f861f77e3c224083b3698e96e3dd160a?auth=74ab0b05a73f42f7177711f7a9787e36#306">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/b5d795fac99442beab307f49935b8a20?auth=db4f6351bdaf4e6eef765fe27163e8b4#289">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>8. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3646514">test_community_message_edit</a>, id: 702843 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/f861f77e3c224083b3698e96e3dd160a?auth=74ab0b05a73f42f7177711f7a9787e36#182">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/b5d795fac99442beab307f49935b8a20?auth=db4f6351bdaf4e6eef765fe27163e8b4#128">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>9. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3646513">test_community_unread_messages_badge</a>, id: 702841 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/f861f77e3c224083b3698e96e3dd160a?auth=74ab0b05a73f42f7177711f7a9787e36#371">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/b5d795fac99442beab307f49935b8a20?auth=db4f6351bdaf4e6eef765fe27163e8b4#350">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestActivityMultipleDevicePRTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3646526">test_activity_center_mentions</a>, id: 702957 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/a83a88b6a5fc426c934dc6cdd30a6c37?auth=955353aed9912e9228aa0d1d24df1cb3#129">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9b603551c8cd497ab908f985b4ed00d7?auth=6ff55648b2652b3eacd17cc3ea429607#101">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3646527">test_activity_center_admin_notification_accept_swipe</a>, id: 702958 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/a83a88b6a5fc426c934dc6cdd30a6c37?auth=955353aed9912e9228aa0d1d24df1cb3#170">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9b603551c8cd497ab908f985b4ed00d7?auth=6ff55648b2652b3eacd17cc3ea429607#117">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityOneDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3646529">test_restore_multiaccount_with_waku_backup_remove_switch</a>, id: 703133 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9578481b869d4d24a17e05c597224c0c?auth=399e038906fce54c8d7068988839195b#230">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3646498">test_community_copy_and_paste_message_in_chat_input</a>, id: 702742 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9578481b869d4d24a17e05c597224c0c?auth=399e038906fce54c8d7068988839195b#77">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3646537">test_community_discovery</a>, id: 703503 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9578481b869d4d24a17e05c597224c0c?auth=399e038906fce54c8d7068988839195b#355">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3646522">test_community_undo_delete_message</a>, id: 702869 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9578481b869d4d24a17e05c597224c0c?auth=399e038906fce54c8d7068988839195b#106">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3646517">test_community_navigate_to_channel_when_relaunch</a>, id: 702846 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9578481b869d4d24a17e05c597224c0c?auth=399e038906fce54c8d7068988839195b#53">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3646533">test_community_mute_community_and_channel</a>, id: 703382 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9578481b869d4d24a17e05c597224c0c?auth=399e038906fce54c8d7068988839195b#125">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestActivityCenterContactRequestMultipleDevicePR:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3646501">test_add_contact_field_validation</a>, id: 702777 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/cf59216a187544ca96b26e18ca3ddde7?auth=5d0294acc485cfaaaf697a9fe4fb5150#140">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/cecdcc344ad7459f91532cac8ea9f203?auth=7266abcc9736f5ecbfef7309dfe798c0#198">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3646519">test_activity_center_contact_request_accept_swipe_mark_all_as_read</a>, id: 702851 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/cf59216a187544ca96b26e18ca3ddde7?auth=5d0294acc485cfaaaf697a9fe4fb5150#105">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/cecdcc344ad7459f91532cac8ea9f203?auth=7266abcc9736f5ecbfef7309dfe798c0#110">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3646518">test_activity_center_contact_request_decline</a>, id: 702850 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/cf59216a187544ca96b26e18ca3ddde7?auth=5d0294acc485cfaaaf697a9fe4fb5150#62">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/cecdcc344ad7459f91532cac8ea9f203?auth=7266abcc9736f5ecbfef7309dfe798c0#56">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details>

--- a/src/quo/components/profile/collectible_list_item/view.cljs
+++ b/src/quo/components/profile/collectible_list_item/view.cljs
@@ -91,7 +91,7 @@
      :color-index     gradient-color-index}]])
 
 (defn- card-details
-  [{:keys [community? avatar-image-src collectible-name theme state set-state]}]
+  [{:keys [community? avatar-image-src collectible-name theme state set-state loading?]}]
   (let [loader-opacity            (reanimated/use-shared-value 1)
         avatar-opacity            (reanimated/use-shared-value 0)
         [load-time set-load-time] (rn/use-state (datetime/now))
@@ -102,7 +102,7 @@
                                                      :avatar-opacity avatar-opacity}))
         empty-name?               (string/blank? collectible-name)]
     (rn/use-mount (fn []
-                    (when (string/blank? avatar-image-src)
+                    (when (and (string/blank? avatar-image-src) (not loading?))
                       (set-avatar-loaded))))
     [rn/view {:style style/card-details-container}
      [reanimated/view {:style (style/avatar-container avatar-opacity)}
@@ -137,7 +137,7 @@
 
 (defn- card-view
   [{:keys [avatar-image-src collectible-name community? counter state set-state
-           gradient-color-index image-src supported-file?]}]
+           gradient-color-index image-src supported-file? loading?]}]
   (let [theme                     (quo.theme/use-theme)
         loader-opacity            (reanimated/use-shared-value (if supported-file? 1 0))
         image-opacity             (reanimated/use-shared-value (if supported-file? 0 1))
@@ -189,6 +189,7 @@
        :community?       community?
        :avatar-image-src avatar-image-src
        :collectible-name collectible-name
+       :loading?         loading?
        :theme            theme}]]))
 
 (defn- image-view
@@ -276,6 +277,7 @@
       [:supported-file? {:optional true} [:maybe boolean?]]
       [:native-ID {:optional true} [:maybe [:or string? keyword?]]]
       [:community? {:optional true} [:maybe boolean?]]
+      [:loading? {:optional true} [:maybe boolean?]]
       [:counter {:optional true} [:maybe [:or :string :int]]]
       [:gradient-color-index {:optional true}
        [:maybe [:enum :gradient-1 :gradient-2 :gradient-3 :gradient-4 :gradient-5]]]

--- a/src/status_im/contexts/wallet/account/tabs/view.cljs
+++ b/src/status_im/contexts/wallet/account/tabs/view.cljs
@@ -25,23 +25,29 @@
   []
   (rf/dispatch [:wallet/request-collectibles-for-current-viewing-account]))
 
-(defn view
-  [{:keys [selected-tab]}]
-  (let [collectible-list        (rf/sub
+(defn- collectibles-tab
+  []
+  (let [updating?               (rf/sub [:wallet/current-viewing-account-collectibles-updating?])
+        collectible-list        (rf/sub
                                  [:wallet/current-viewing-account-collectibles-in-selected-networks])
         current-account-address (rf/sub [:wallet/current-viewing-account-address])]
-    [rn/view {:style {:flex 1}}
-     (case selected-tab
-       :assets       [assets/view]
-       :collectibles [collectibles/view
-                      {:collectibles              collectible-list
-                       :current-account-address   current-account-address
-                       :on-end-reached            on-end-reached
-                       :on-collectible-press      on-collectible-press
-                       :on-collectible-long-press on-collectible-long-press}]
-       :activity     [activity/view]
-       :permissions  [empty-tab/view
-                      {:title        (i18n/label :t/no-permissions)
-                       :description  (i18n/label :t/no-collectibles-description)
-                       :placeholder? true}]
-       [about/view])]))
+    [collectibles/view
+     {:loading?                  updating?
+      :collectibles              collectible-list
+      :current-account-address   current-account-address
+      :on-end-reached            on-end-reached
+      :on-collectible-press      on-collectible-press
+      :on-collectible-long-press on-collectible-long-press}]))
+
+(defn view
+  [{:keys [selected-tab]}]
+  [rn/view {:style {:flex 1}}
+   (case selected-tab
+     :assets       [assets/view]
+     :collectibles [collectibles-tab]
+     :activity     [activity/view]
+     :permissions  [empty-tab/view
+                    {:title        (i18n/label :t/no-permissions)
+                     :description  (i18n/label :t/no-collectibles-description)
+                     :placeholder? true}]
+     [about/view])])

--- a/src/status_im/contexts/wallet/collectible/events.cljs
+++ b/src/status_im/contexts/wallet/collectible/events.cljs
@@ -1,11 +1,15 @@
 (ns status-im.contexts.wallet.collectible.events
   (:require [camel-snake-kebab.extras :as cske]
+            [clojure.set]
             [clojure.string :as string]
             [react-native.platform :as platform]
             [status-im.contexts.network.data-store :as network.data-store]
             [status-im.contexts.wallet.collectible.utils :as collectible-utils]
+            [status-im.contexts.wallet.data-store :as data-store]
             [taoensso.timbre :as log]
+            [utils.collection]
             [utils.ethereum.chain :as chain]
+            [utils.number :as utils.number]
             [utils.re-frame :as rf]
             [utils.transforms :as transforms]))
 
@@ -23,6 +27,12 @@
 
 (def max-cache-age-seconds 3600)
 (def collectibles-request-batch-size 25)
+
+(def ownership-state
+  {:idle     1
+   :delayed  2
+   :updating 3
+   :error    4})
 
 (defn- move-collectibles-to-accounts
   [accounts new-collectibles-per-account]
@@ -84,12 +94,18 @@
 (rf/reg-event-fx
  :wallet/request-collectibles-for-all-accounts
  (fn [{:keys [db]} [{:keys [new-request?]}]]
-   (let [accounts                 (->> (get-in db [:wallet :accounts])
+   (let [updating-addresses       (-> (get-in db [:wallet :ui :collectibles :updating])
+                                      keys
+                                      set)
+         accounts                 (->> (get-in db [:wallet :accounts])
                                        (filter (fn [[_ {:keys [has-more-collectibles?]}]]
                                                  (or (nil? has-more-collectibles?)
                                                      (true? has-more-collectibles?))))
                                        (keys))
-         num-accounts             (count accounts)
+         ;; filter the addresses which are requested before and the collectibles are updating
+         requestable-addresses    (clojure.set/difference (set accounts)
+                                                          updating-addresses)
+         num-accounts             (count requestable-addresses)
          collectibles-per-account (quot collectibles-request-batch-size num-accounts)
          ;; We need to pass unique IDs for simultaneous requests, otherwise they'll fail
          request-ids              (get-unique-collectible-request-id num-accounts)
@@ -100,13 +116,13 @@
                                             :account    account
                                             :amount     collectibles-per-account}]])
                                        request-ids
-                                       accounts)]
+                                       requestable-addresses)]
      {:db (cond-> db
             :always      (assoc-in [:wallet :ui :collectibles :pending-requests] num-accounts)
             new-request? (update-in [:wallet :accounts] update-vals #(dissoc % :collectibles)))
       :fx collectible-requests})))
 
-(defn request-new-collectibles-for-account-from-signal
+(defn request-collectibles-for-account
   [{:keys [db]} [address]]
   (let [pending-requests (get-in db [:wallet :ui :collectibles :pending-requests] 0)
         [request-id]     (get-unique-collectible-request-id 1)]
@@ -117,49 +133,138 @@
              :account    address
              :amount     collectibles-request-batch-size}]]]}))
 
-(rf/reg-event-fx :wallet/request-new-collectibles-for-account-from-signal
- request-new-collectibles-for-account-from-signal)
+(rf/reg-event-fx :wallet/request-collectibles-for-account request-collectibles-for-account)
 
 (rf/reg-event-fx
  :wallet/request-collectibles-for-current-viewing-account
  (fn [{:keys [db]} _]
    (when (network.data-store/online? db)
-     (let [current-viewing-account (-> db :wallet :current-viewing-account-address)
-           [request-id]            (get-unique-collectible-request-id 1)]
-       {:db (assoc-in db [:wallet :ui :collectibles :pending-requests] 1)
-        :fx [[:dispatch
-              [:wallet/request-new-collectibles-for-account
-               {:request-id request-id
-                :account    current-viewing-account
-                :amount     collectibles-request-batch-size}]]]}))))
+     (let [current-viewing-account (-> db :wallet :current-viewing-account-address)]
+       {:fx [[:dispatch [:wallet/request-collectibles-for-account current-viewing-account]]]}))))
+
+(rf/reg-event-fx
+ :wallet/collectible-ownership-update-finished-with-error
+ (fn [{:keys [db]} [{:keys [accounts message chainId]}]]
+   (let [address            (first accounts)
+         pending-chain-ids  (get-in db [:wallet :ui :collectibles :updating address])
+         updated-chain-ids  (disj pending-chain-ids chainId)
+         all-chain-updated? (and (some? pending-chain-ids) (empty? updated-chain-ids))]
+     {:db (cond-> db
+            (some? pending-chain-ids)
+            (assoc-in [:wallet :ui :collectibles :updating address] updated-chain-ids))
+      :fx [[:dispatch
+            [:wallet/log-rpc-error
+             {:event  :wallet/collectible-ownership-update-finished-with-error
+              :params {:address  address
+                       :chain-id chainId}}
+             message]]
+           (when all-chain-updated?
+             [:dispatch [:wallet/request-collectibles-for-account address]])]})))
+
+(rf/reg-event-fx
+ :wallet/collectible-ownership-update-finished
+ (fn [{:keys [db]} [{:keys [accounts chainId]}]]
+   (let [address            (first accounts)
+         pending-chain-ids  (get-in db [:wallet :ui :collectibles :updating address])
+         updated-chain-ids  (disj pending-chain-ids chainId)
+         all-chain-updated? (and (some? pending-chain-ids) (empty? updated-chain-ids))]
+     {:db (cond-> db
+            (some? pending-chain-ids)
+            (assoc-in [:wallet :ui :collectibles :updating address] updated-chain-ids))
+      :fx [(when all-chain-updated?
+             [:dispatch [:wallet/request-collectibles-for-account address]])]})))
+
+(defn- update-collectibles-in-account
+  [existing-collectibles updated-collectibles]
+  (let [indexed-existing (utils.collection/index-by
+                          :unique-id
+                          existing-collectibles)
+        existing-ids     (-> indexed-existing keys vec)
+        ;; pick collectibles only in the app-db
+        indexed-updated  (-> (utils.collection/index-by
+                              :unique-id
+                              updated-collectibles)
+                             (select-keys existing-ids))]
+    (-> (merge-with
+         merge
+         indexed-existing
+         indexed-updated)
+        vals
+        vec)))
+
+(rf/reg-event-fx
+ :wallet/collectibles-data-updated
+ (fn [{:keys [db]} [{:keys [message]}]]
+   (let [collectibles-by-address (->> message
+                                      transforms/json->clj
+                                      data-store/rpc->collectibles
+                                      (group-by #(-> % :ownership first :address)))]
+     {:db (update-in db
+                     [:wallet :accounts]
+                     #(reduce-kv
+                       (fn [accounts address updated-collectibles]
+                         (if (contains? accounts address)
+                           (update-in accounts
+                                      [address :collectibles]
+                                      update-collectibles-in-account
+                                      updated-collectibles)
+                           accounts))
+                       %
+                       collectibles-by-address))})))
+
+(rf/reg-event-fx
+ :wallet/set-collectibles-updating-status
+ (fn [{:keys [db]} [address chain-ids]]
+   {:db (assoc-in db [:wallet :ui :collectibles :updating address] chain-ids)}))
 
 (defn- update-fetched-collectibles-progress
   [db owner-address collectibles offset has-more?]
   (-> db
+      (update-in [:wallet :ui :collectibles :updating] dissoc owner-address)
       (assoc-in [:wallet :ui :collectibles :fetched owner-address] collectibles)
       (assoc-in [:wallet :accounts owner-address :current-collectible-idx]
                 (+ offset (count collectibles)))
       (assoc-in [:wallet :accounts owner-address :has-more-collectibles?] has-more?)))
 
+(defn- updating-collectibles?
+  [status]
+  (and (= (:state status) (ownership-state :updating))
+       (= (:timestamp status) -1)))
+
 (rf/reg-event-fx
  :wallet/owned-collectibles-filtering-done
  (fn [{:keys [db]} [{:keys [message]}]]
    (let [{:keys [offset ownershipStatus collectibles
-                 hasMore]} (transforms/json->clj message)
-         collectibles      (cske/transform-keys transforms/->kebab-case-keyword collectibles)
-         pending-requests  (dec (get-in db [:wallet :ui :collectibles :pending-requests]))
-         owner-address     (some->> ownershipStatus
-                                    first
-                                    key
-                                    name)]
+                 hasMore]}           (transforms/json->clj message)
+         ownership-status-by-address (-> ownershipStatus
+                                         (update-keys name)
+                                         (update-vals #(update-keys %
+                                                                    (comp utils.number/parse-int name))))
+         owner-address               (some-> ownership-status-by-address
+                                             first
+                                             key)
+         ownership-status            (get ownership-status-by-address owner-address)
+         collectibles                (data-store/rpc->collectibles collectibles)
+         pending-requests            (dec (get-in db [:wallet :ui :collectibles :pending-requests]))
+         ;; check if collectibles are updating (never fetched and cached before) for this address
+         updating-chains             (-> (select-keys
+                                          ownership-status
+                                          (for [[k v] ownership-status
+                                                :when (updating-collectibles? v)]
+                                            k))
+                                         keys
+                                         set)
+         updating?                   (-> updating-chains count pos?)]
      {:db (cond-> db
             :always       (assoc-in [:wallet :ui :collectibles :pending-requests] pending-requests)
             owner-address (update-fetched-collectibles-progress owner-address
                                                                 collectibles
                                                                 offset
-                                                                hasMore))
+                                                                (when-not updating? hasMore)))
       :fx [(when (zero? pending-requests)
-             [:dispatch [:wallet/flush-collectibles-fetched]])]})))
+             [:dispatch [:wallet/flush-collectibles-fetched]])
+           (when updating?
+             [:dispatch [:wallet/set-collectibles-updating-status owner-address updating-chains]])]})))
 
 (rf/reg-event-fx
  :wallet/navigate-to-collectible-details

--- a/src/status_im/contexts/wallet/collectible/events_test.cljs
+++ b/src/status_im/contexts/wallet/collectible/events_test.cljs
@@ -43,8 +43,8 @@
 
         (is (match? result-db expected-db))))))
 
-(deftest request-new-collectibles-for-account-from-signal-test
-  (testing "request new collectibles for account from signal"
+(deftest request-collectibles-for-account-test
+  (testing "request collectibles for account"
     (let [db       {:wallet {}}
           address  "0x1"
           expected {:db {:wallet {:ui {:collectibles {:pending-requests 1}}}}
@@ -53,6 +53,6 @@
                            {:request-id 0
                             :account    address
                             :amount     events/collectibles-request-batch-size}]]]}
-          effects  (events/request-new-collectibles-for-account-from-signal {:db db}
-                                                                            [address])]
+          effects  (events/request-collectibles-for-account {:db db}
+                                                            [address])]
       (is (match? expected effects)))))

--- a/src/status_im/contexts/wallet/collectible/utils.cljs
+++ b/src/status_im/contexts/wallet/collectible/utils.cljs
@@ -70,3 +70,10 @@
                                                        test-networks-enabled?
                                                        is-goerli-enabled?)]
     (str base-link "/assets/" opensea-network-name "/" contract-address "/" token-id)))
+
+(defn get-collectible-unique-id
+  [{:keys [id]}]
+  (let [chain-id         (-> id :contract-id :chain-id)
+        contract-address (-> id :contract-id :address)
+        token-id         (-> id :token-id)]
+    (str chain-id contract-address token-id)))

--- a/src/status_im/contexts/wallet/common/collectibles_tab/view.cljs
+++ b/src/status_im/contexts/wallet/common/collectibles_tab/view.cljs
@@ -10,6 +10,16 @@
     [utils.i18n :as i18n]
     [utils.re-frame :as rf]))
 
+(defn- loading-collectible-item
+  [_ index]
+  (let [gradient-color (keyword (str "gradient-" (inc (mod index 5))))]
+    [quo/collectible-list-item
+     {:type                 :card
+      :gradient-color-index gradient-color
+      :supported-file?      true
+      :loading?             true
+      :container-style      style/collectible-container}]))
+
 (defn- collectible-item
   [{:keys [preview-url collection-data collectible-data total-owned on-press on-long-press]
     :as   collectible}
@@ -41,10 +51,18 @@
 
 (defn view
   [{:keys [collectibles filtered? on-end-reached on-collectible-press
-           current-account-address on-collectible-long-press]}]
+           current-account-address on-collectible-long-press loading?]}]
   (let [theme                   (quo.theme/use-theme)
         no-results-match-query? (and filtered? (empty? collectibles))]
     (cond
+      loading?
+      [rn/flat-list
+       {:data                    (repeat 8 {})
+        :style                   {:flex 1}
+        :content-container-style style/list-container-style
+        :num-columns             2
+        :render-fn               loading-collectible-item}]
+
       no-results-match-query?
       [rn/view {:style {:flex 1 :justify-content :center}}
        [quo/empty-state
@@ -62,15 +80,14 @@
       ;; TODO: https://github.com/status-im/status-mobile/issues/20137
       ;; 1. If possible, move `collectibles-data` calculation to a subscription
       ;; 2. Optimization: do not recalculate all the collectibles, process only the new ones
-      (let [collectibles-data (map-indexed
-                               (fn [index {:keys [ownership] :as collectible}]
+      (let [collectibles-data (map
+                               (fn [{:keys [ownership] :as collectible}]
                                  (let [total-owned (rf/sub [:wallet/total-owned-collectible ownership
                                                             current-account-address])]
                                    (assoc collectible
-                                          :total-owned       total-owned
-                                          :on-long-press     on-collectible-long-press
-                                          :on-press          on-collectible-press
-                                          :collectible-index index)))
+                                          :total-owned   total-owned
+                                          :on-long-press on-collectible-long-press
+                                          :on-press      on-collectible-press)))
                                collectibles)]
         [rn/flat-list
          {:data                     collectibles-data
@@ -80,5 +97,5 @@
           :num-columns              2
           :render-fn                collectible-item
           :on-end-reached           on-end-reached
-          :key-fn                   :collectible-index
+          :key-fn                   :unique-id
           :on-end-reached-threshold 4}]))))

--- a/src/status_im/contexts/wallet/data_store.cljs
+++ b/src/status_im/contexts/wallet/data_store.cljs
@@ -4,6 +4,7 @@
     [clojure.set :as set]
     [clojure.string :as string]
     [status-im.constants :as constants]
+    [status-im.contexts.wallet.collectible.utils :as collectible-utils]
     [status-im.contexts.wallet.common.utils.networks :as network-utils]
     [status-im.contexts.wallet.send.utils :as send-utils]
     [utils.collection :as utils.collection]
@@ -266,3 +267,10 @@
 (defn tokens-never-loaded?
   [db]
   (nil? (get-in db [:wallet :ui :tokens-loading])))
+
+(defn rpc->collectibles
+  [collectibles]
+  (->> collectibles
+       (cske/transform-keys transforms/->kebab-case-keyword)
+       (map #(assoc % :unique-id (collectible-utils/get-collectible-unique-id %)))
+       vec))

--- a/src/status_im/contexts/wallet/events.cljs
+++ b/src/status_im/contexts/wallet/events.cljs
@@ -105,7 +105,7 @@
  :wallet/fetch-assets-for-address
  (fn [_ [address]]
    {:fx [[:dispatch [:wallet/get-wallet-token-for-account address]]
-         [:dispatch [:wallet/request-new-collectibles-for-account-from-signal address]]
+         [:dispatch [:wallet/request-collectibles-for-account address]]
          [:dispatch [:wallet/check-recent-history-for-account address]]]}))
 
 (defn- reconcile-accounts
@@ -150,11 +150,6 @@
           [{:method     "accounts_getAccounts"
             :on-success [:wallet/get-accounts-success]
             :on-error   [:wallet/log-rpc-error {:event :wallet/get-accounts}]}]]]}))
-
-(rf/reg-event-fx :wallet/process-account-from-signal
- (fn [{:keys [db]} [{:keys [address] :as account}]]
-   {:db (assoc-in db [:wallet :accounts address] (data-store/rpc->account account))
-    :fx [[:dispatch [:wallet/fetch-assets-for-address address]]]}))
 
 (rf/reg-event-fx
  :wallet/save-account
@@ -584,25 +579,13 @@
 
 (rf/reg-event-fx
  :wallet/process-keypair-from-backup
- (fn [{:keys [db]} [{:keys [backedUpKeypair]}]]
-   (let [{:keys [key-uid accounts]} backedUpKeypair
-         accounts-fx
-         (mapv (fn [{:keys [chat] :as account}]
-                 ;; We exclude the chat account from the profile keypair for fetching the assets
-                 (when-not chat
-                   [:dispatch
-                    [:wallet/process-account-from-signal
-                     account]]))
-               accounts)]
-     {:db (assoc-in db
-           [:wallet :keypairs key-uid]
-           (data-store/rpc->keypair backedUpKeypair))
-      :fx accounts-fx})))
+ (fn [_ [{:keys [backedUpKeypair]}]]
+   {:fx [[:dispatch [:wallet/reconcile-keypairs [backedUpKeypair]]]]}))
 
 (rf/reg-event-fx
  :wallet/process-watch-only-account-from-backup
  (fn [_ [{:keys [backedUpWatchOnlyAccount]}]]
-   {:fx [[:dispatch [:wallet/process-account-from-signal backedUpWatchOnlyAccount]]]}))
+   {:fx [[:dispatch [:wallet/reconcile-watch-only-accounts [backedUpWatchOnlyAccount]]]]}))
 
 (defn reconcile-watch-only-accounts
   [{:keys [db]} [watch-only-accounts]]

--- a/src/status_im/contexts/wallet/events_test.cljs
+++ b/src/status_im/contexts/wallet/events_test.cljs
@@ -143,13 +143,6 @@
                              address-2 {:address address-2}}}})
       (is (match? expected-fx (:fx (dispatch [event-id])))))))
 
-(h/deftest-event :wallet/process-account-from-signal
-  [event-id dispatch]
-  (let [expected-effects {:db {:wallet {:accounts {address account}}}
-                          :fx [[:dispatch [:wallet/fetch-assets-for-address address]]]}]
-    (reset! rf-db/app-db {:wallet {:accounts {}}})
-    (is (match? expected-effects (dispatch [event-id raw-account])))))
-
 (h/deftest-event :wallet/reconcile-keypairs
   [event-id dispatch]
   (let [keypair-key-uid (:key-uid raw-account)]

--- a/src/status_im/contexts/wallet/home/tabs/view.cljs
+++ b/src/status_im/contexts/wallet/home/tabs/view.cljs
@@ -19,17 +19,23 @@
 
 (def on-collectible-press #(rf/dispatch [:wallet/navigate-to-collectible-details %]))
 
+(def request-collectibles #(rf/dispatch [:wallet/request-collectibles-for-all-accounts {}]))
+
+(defn- collectibles-tab
+  []
+  (let [updating?        (rf/sub [:wallet/home-tab-collectibles-updating?])
+        collectible-list (rf/sub [:wallet/owned-collectibles-list-in-selected-networks])]
+    [collectibles/view
+     {:loading?                  updating?
+      :collectibles              collectible-list
+      :on-collectible-long-press on-collectible-long-press
+      :on-end-reached            request-collectibles
+      :on-collectible-press      on-collectible-press}]))
+
 (defn view
   [{:keys [selected-tab]}]
-  (let [collectible-list     (rf/sub [:wallet/owned-collectibles-list-in-selected-networks])
-        request-collectibles #(rf/dispatch
-                               [:wallet/request-collectibles-for-all-accounts {}])]
-    [rn/view {:style style/container}
-     (case selected-tab
-       :assets       [assets/view]
-       :collectibles [collectibles/view
-                      {:collectibles              collectible-list
-                       :on-collectible-long-press on-collectible-long-press
-                       :on-end-reached            request-collectibles
-                       :on-collectible-press      on-collectible-press}]
-       [activity/view {:activities []}])]))
+  [rn/view {:style style/container}
+   (case selected-tab
+     :assets       [assets/view]
+     :collectibles [collectibles-tab]
+     [activity/view {:activities []}])])

--- a/src/status_im/contexts/wallet/signals.cljs
+++ b/src/status_im/contexts/wallet/signals.cljs
@@ -29,6 +29,21 @@
           [:wallet/pending-transaction-status-changed-received
            (transforms/js->clj event-js)]]]}
 
+       "wallet-collectibles-ownership-update-finished"
+       {:fx [[:dispatch
+              [:wallet/collectible-ownership-update-finished
+               (transforms/js->clj event-js)]]]}
+
+       "wallet-collectibles-ownership-update-finished-with-error"
+       {:fx [[:dispatch
+              [:wallet/collectible-ownership-update-finished-with-error
+               (transforms/js->clj event-js)]]]}
+
+       "wallet-collectibles-data-updated"
+       {:fx [[:dispatch
+              [:wallet/collectibles-data-updated
+               (transforms/js->clj event-js)]]]}
+
        "wallet-owned-collectibles-filtering-done"
        {:fx [[:dispatch
               [:wallet/owned-collectibles-filtering-done

--- a/src/status_im/subs/wallet/collectibles.cljs
+++ b/src/status_im/subs/wallet/collectibles.cljs
@@ -137,3 +137,29 @@
                  acc))
              0
              ownership))))
+
+(re-frame/reg-sub
+ :wallet/collectibles
+ :<- [:wallet/ui]
+ :-> :collectibles)
+
+(re-frame/reg-sub
+ :wallet/collectibles-updating
+ :<- [:wallet/collectibles]
+ :-> :updating)
+
+(re-frame/reg-sub
+ :wallet/current-viewing-account-collectibles-updating?
+ :<- [:wallet/collectibles-updating]
+ :<- [:wallet/current-viewing-account-address]
+ (fn [[updating-addresses address]]
+   (contains? updating-addresses address)))
+
+(re-frame/reg-sub
+ :wallet/home-tab-collectibles-updating?
+ :<- [:wallet/collectibles-updating]
+ :<- [:wallet/accounts-without-watched-accounts]
+ (fn [[updating-addresses accounts]]
+   (->> accounts
+        (map :address)
+        (every? #(contains? updating-addresses %)))))

--- a/status-go-version.json
+++ b/status-go-version.json
@@ -3,7 +3,7 @@
     "_comment": "Instead use: scripts/update-status-go.sh <rev>",
     "owner": "status-im",
     "repo": "status-go",
-    "version": "fix/send-account-event-on-handling-backedup-accounts",
-    "commit-sha1": "e34b2fb1a10dc25ed5385b9fae78cc6a18b06410",
-    "src-sha256": "1f72zwpq7s5fd89dp25l9pib04kzgbyy46imbn765pcszdabcc04"
+    "version": "v0.184.52",
+    "commit-sha1": "81cfce709e8d123eb1956b87f8e0f19dc47c122c",
+    "src-sha256": "0hna30ms4ccxmi20l5v36y84pva1s4jjkqg11whwmdjgw75d0fan"
 }

--- a/status-go-version.json
+++ b/status-go-version.json
@@ -3,7 +3,7 @@
     "_comment": "Instead use: scripts/update-status-go.sh <rev>",
     "owner": "status-im",
     "repo": "status-go",
-    "version": "v0.184.51",
-    "commit-sha1": "9e722ed09544cc798bc8aa5cdb3b7b58241fbf95",
-    "src-sha256": "1r8w9309gffb52myfgnqyv8cpn372qkq14hpwk2zmz86qphw9jvf"
+    "version": "fix/send-account-event-on-handling-backedup-accounts",
+    "commit-sha1": "e34b2fb1a10dc25ed5385b9fae78cc6a18b06410",
+    "src-sha256": "1f72zwpq7s5fd89dp25l9pib04kzgbyy46imbn765pcszdabcc04"
 }

